backend/src/util/mapping/Mappings.js

```diff
@@ -366,25 +351,30 @@ const PermissionMappings = [
   { back: 'ofm_portal_privilege_number', front: 'permissionNumber' },
 ]
 
+// const ReportTemplateMappings = [
```

Comment:
I don't know that there's any need to comment this out given that it's not API code. Fine as is though.

Reply:
let's keep it this way just in case we need to remove all of them in the future.